test/scale-infra/roles/ec2instances/README.md

```diff
@@ -0,0 +1,58 @@
+ec2instances
+=========
+
+Manage EC2 instance(s).
+
+Requirements
+------------
+
+Python package `boto` is required.
+
+Role Variables
+--------------
+
+AWS credentials are NOT coded as role variables. They can be set by environment variables, e.g. `AWS_ACCESS_KEY` and `AWS_SECRET_KEY`.
```

Comment:
Environment variables are the least secure way to pass confidential information around. Could another way be used?

Reply:
They are needed for the AWS CLI - the env variables are just one of the options to provide the credentials into the CLI.

Reply:
To improve the instructions, I just replaced that section with this new one: https://github.com/dumb0002/kubestellar/tree/scale-test-infra/test/scale-infra#prerequisites